lms/static/scripts/frontend_apps/components/test/LaunchErrorDialog-test.js

```diff
@@ -65,7 +65,7 @@ describe('LaunchErrorDialog', () => {
     {
       errorState: 'canvas_group_set_not_found',
       expectedText:
-        "Assignment's group set no longer exists in CanvasHypothesis couldn't load this assignment because the assignment's group set no longer exists in Canvas.To fix this problem, an instructor needs to edit the assignment settings and select a new group set.Detailed error info.If the problem persists, send us an email or open a support ticket. You can also visit our help documents.",
+        "Assignment's group set no longer exists in CanvasHypothesis couldn't load this assignment because the assignment's group set no longer exists in Canvas.To fix this problem, an instructor needs to edit the assignment settings and select a new group set.",
```

Comment:
Man, these particular tests are fiddly.

Reply:
Having one really long string here is a bit silly. It would probably make more sense for `expectedText` to be an array of strings and then check for the presence of each one.

I assume these changes are required because a component that was not previously being mocked now is with the new version of `mockImportedComponents`?

Reply:
Yes, these tests could use refactor. 

> I assume these changes are required because a component that was not previously being mocked now is with the new version of mockImportedComponents?

That's my assumption. The change to this `mockImportedComponents` included a change in how components are identified (that is, a change beyond the don't-mock-external-components bit), and it seems plausible that a formerly-not-mocked component now _is_ mocked.